### ISSUE TITLE
Banning trackers like Google Analytics

### ISSUE THREAD

**Issue by realpixelcode** [raisely/NoHarm#75]
## Overview

Trackers are used on websites and in mobile apps to spy on the user and to find out what other internet services he uses, based on cookies or fingerprinting. This is obviously highly privacy-invading, which is the reason why, for example, many privacy-concious people use [uBlock Origin](https://github.com/gorhill/uBlock) and why tracking without consent is prohibited in the EU by the GDPR.

## Proposed Resolution

Therefore, I propose that tracking is banned. For reference, here's the For Good Eyes Only Licence's [definition of trackers](https://forgoodeyesonly.codeberg.page/v0.2/#trackers):

> Trackers are software designed to
>
>  1. collect information about a user or mark him/her in such a way that
>     he/she can be almost reliably recognised at a later point in time or
>     across different websites or (digital) services or to
>
>  2. derive identifying data points about a user’s personal preferences,
>     behaviour or other personal characteristics,
>
> without this being immediately imperative and thus justified by the absolute
> core function of the overall software.

**Comment by ghost** [comment]
*(edited for clarity)*

Although I agree about how creepy trackers are @realpixelcode, schools and learning management systems (ex. Google Classroom) use trackers to to make sure that students are:
1. Not on inappropriate websites.
2. Not looking up how to do things to cause harm (ex. searching "How to make a bomb" on Google; yes, this really happened BTW :disappointed:).
3. Not wasting time on games.
4. Staying on task.

**Comment by realpixelcode** [comment]
This absolutely sounds like `being immediately imperative and thus justified` though.

**Comment by tommaitland** [comment]
When we've been designing this license we've needed to strike a balance between pure idealism and outlining all of our beliefs for a better world, and ensuring the license is pragmatic and adoptable.

Google Analytics is on something like 70% of the web (not sure where I got that from, but have seen it somewhere) and often tracking is required to support revenue. For example if you rely on Google Ads to grow your business, you're going to have a really hard time not wasting loads of money on Google Ads without Google Analytics.

I'm aligned and I know a lot of others who have worked on this license at Raisely are aligned that invasive tracking is bad, but I believe banning it from this license would render the license fairly inoperable for the modern web.
